### ISSUE TITLE
ahume/go-github is behind and failing

### ISSUE THREAD

**Issue by mattdodge** [ahume/github-deployment-resource#11]
Trying to build/test from source and seeing this error:
```
./github.go:58:47: g.client.Repositories.GetDeployment undefined (type *github.RepositoriesService has no field or method GetDeployment)
```

It looks like what is committed to github.com/ahume/go-github is behind what's on your local machine maybe @ahume? At this point though google's go-github module is way far ahead and breaking way more than just this one error.

Can you confirm and push what you have or maybe provide some other guidance on how to proceed? Would like to make a PR here but am hung up on this for now.

**Comment by ahume** [comment]
@mattdodge I think/hope I've resolved this with #12 . Updated to the latest google/go-github and converted dependency management to modules.

Let me know if this unblocks you. I'm going to work on some of the other pull requests that are currently open, as well as improve the test/CI aspects of this repo.

**Comment by mattdodge** [comment]
Yep, looks great, thanks!
